lib/icinga/apievents.cpp

```diff
@@ -182,7 +182,7 @@ void ApiEvents::FlappingChangedHandler(const Checkable::Ptr& checkable, const Me
 
 void ApiEvents::AcknowledgementSetHandler(const Checkable::Ptr& checkable,
 	const String& author, const String& comment, AcknowledgementType type,
-	bool notify, bool persistent, double expiry, const MessageOrigin::Ptr& origin)
+	bool notify, bool persistent, double, double expiry, const MessageOrigin::Ptr& origin)
```

Comment:
Where's the parameter? Shouldn't we declare and use that also in our REST API?

Reply:
IMAO it's an internal only info. In our ReST API we have `Checkable#acknowledgement_last_change` now.